packages/app/package.json

```diff
@@ -10,18 +10,19 @@
     "tsc": "tsc"
   },
   "dependencies": {
+    "@ant-design/charts": "^1.0.11",
```

Comment:
if this isn't working, maybe get rid of it from package.json

Reply:
yeah unfortunately we could not get it working so I removed it. 

I still think there's hope, but we can explore again in the future